doc/JSON_INFO.md

```diff
@@ -1887,6 +1891,27 @@ Vehicle components when installed on a vehicle.
 },
 ```
 
+#### Symbols and Variants
+Vehicle parts can have multiple identical variants that use different symbols (and potentially
+tileset sprites).  They are declared by the `"standard_symbols"` boolean or the "symbols" object.
+Variants are used in the vehicle prototype as a suffix following the part id (ie `id_variant`), such as `"frame_nw"` or `"halfboard_cover"`.
+
+setting `"standard_symbols"` to true gives the vehicle the following variants:
+```
+"cover": "^", "cross": "c", "horizontal": "h", "horizontal_2": "=", "vertical": "j",
+"vertical_2": "H", "ne": "u", "nw": "y", "se": "n", "sw": "b"
+```
+
+Otherwise, variants can use any of the following suffices:
+```
+"cover", "cross", "horizontal_front", "horizontal_front_edge", "horizontal_rear",
+"horizontal_rear_edge", "horizontal_2_front", "horizontal_2_rear", "ne_edge", "nw_edge",
+"se_edge", "sw_edge", "vertical_right", "vertical_left", "vertical_2_right", "vertical_2_left",
+"vertical_T_right", "vertical_T_left", front_left", "front_right", "rear_left", rear_right",
+"vertical", "horizontal", "vertical_2", "horizontal_2", "ne", "nw", "se", "sw", "front", "rear",
+"left," "right"
```

Comment:
Are these hard-coded? We have got a few parts not yet added that use different names, such as frame_horizontal_left and frame_horizontal_2_left. I could use different names, like vertical_t_l/r for horizontal_l/r.

Reply:
I would really prefer less names and hardcoded names.